### ISSUE TITLE
Consider bin script as a package usage

### ISSUE THREAD

**Issue by brunolemos** [dylang/npm-check#237]
> **extensible-react-scripts** 😕  NOTUSED? 
Still using extensible-react-scripts?
Depcheck did not find code similar to `require('extensible-react-scripts')`
or `import from 'extensible-react-scripts'`

I indeed don't do imports from this package, but I use it on npm scripts. e.g:
```js
"start": "extensible-react-scripts start",
"test": "extensible-react-scripts test --env=jsdom"
```

Could that be considered a package usage?

**Comment by scottmas** [comment]
Yes. This makes the `NOTUSED?` flag much less useful than it could be.

**Comment by Ionaru** [comment]
+1 for this issue.

I am using ts-node like this: `"start": "node  --require ts-node/register src/index.ts"`, it'd be great if npm-check would consider ts-node as used.

**Comment by LinusU** [comment]
Yeah, this bites me with `mocha` and `standard` for every package I publish 😄 

Would love to accept a pull request, any takers? 💌

**Comment by Ionaru** [comment]
Looks like this will need to be fixed in https://github.com/depcheck/depcheck, that's what this package uses to determine unused packages.

**Comment by Ionaru** [comment]
@LinusU Can you post the scripts you use `mocha` and `standard` in? It would help me test a fix.

**Comment by LinusU** [comment]
@Ionaru 

```js
{
  // ...
  "scripts": {
    "test": "standard && mocha"
  }
}
```

Thanks for getting on top of this ❤️

**Comment by Ionaru** [comment]
@LinusU I copy-pasted that script into my test setup, but `npm-check` does not see them as `unused`.

**Comment by LinusU** [comment]
Hmm, have you also added `standard` and `mocha` to your dependencies? `npm install --save-dev standard mocha`

**Comment by Ionaru** [comment]
Yes, I installed them

**Comment by jkubecki** [comment]
When I use depcheck by itself, I use a .depcheckrc to include my bin folder explicitly:

```yaml
parsers:
  '**/*.js': ['es6']
  '**/bin/*': ['es6']
```

What would be _nice_ would be if npm-check would also "honor" the presence of a .depcheckrc or other depcheck config file?
